debian/changelog

```diff
@@ -1,3 +1,10 @@
+cloud-init (22.1-14-g2e17a0d6-0ubuntu1~22.04.4) jammy; urgency=medium
+
+  * cherry-pick eee60329: Fix cloud-init status --wait when no datasource
```

Comment:
This isn't a "you have to redo this" level feedback, but FYI you'd usually reference any new patch added in debian/patches here.

Two common styles are
```
* d/p/cpick-eee60329-Fix-cloud-init-status-wait-when-no-datasource-found:
  cherry-pick eee60329: Fix cloud-init status --wait when no datasource
  found (#1349)
```

```
* cherry-pick eee60329: Fix cloud-init status --wait when no datasource
  found (#1349)
  - d/p/cpick-eee60329-Fix-cloud-init-status-wait-when-no-datasource-found 
```

The latter is more common if you have a lift of patches picked for one overall change/fix.